src/dmd/lexer.d

```diff
@@ -1130,19 +1330,24 @@ class Lexer : ErrorHandler
      */
     final uint escapeSequence()
     {
-        return Lexer.escapeSequence(this, p);
+        return Lexer.escapeSequence(token.loc, diagnosticReporter, p);
     }
 
     /**
     Parse the given string literal escape sequence into a single character.
     Params:
-        handler = the error handler object
+        handler = the diagnostic reporter object
         sequence = pointer to string with escape sequence to parse. this is a reference
                    variable that is also used to return the position after the sequence
     Returns:
         the escaped sequence as a single character
     */
-    static dchar escapeSequence(ErrorHandler handler, ref const(char)* sequence)
+    static dchar escapeSequence(const ref Loc loc, DiagnosticReporter handler, ref const(char)* sequence)
```

Comment:
```
dmd/lexer.d(1345): Warning: Ddoc: parameter count mismatch, expected 3, got 2
```

tl;dr: `loc` also needs to be documented

Reply:
Yes, I noticed.

Reply:
Fixed.